backend/common/settings-access.js

```diff
@@ -146,6 +146,15 @@ settings.backupOnExit = function() {
     return backupOnExit != null ? backupOnExit : true;
 };
 
+settings.backupIncludeResources = function() {
+    const save = getDataFromFile("/settings/backupIncludeResources");
+    return save != null ? save : true;
+};
+
+settings.setBackupIncludeResources = function(backupIncludeResources) {
+    pushDataToFile("/settings/backupIncludeResources", backupIncludeResources === true);
```

Comment:
We'll probably want to default to off so that current functionality isn't changed.

Reply:
that is how it currently works.  you must uncheck the "Include Resources" to turn it off as the current functionality is to include them.

Reply:
> We'll probably want to default to off so that current functionality isn't changed.

the verbage here  would you prefer include or ignore?